Configurations/10-main.conf

```diff
@@ -6,18 +6,21 @@ my $vc_win64a_info = {};
 sub vc_win64a_info {
     unless (%$vc_win64a_info) {
         if (`nasm -v 2>NUL` =~ /NASM version ([0-9]+\.[0-9]+)/ && $1 >= 2.0) {
-            $vc_win64a_info = { as        => "nasm",
-                                asflags   => "-f win64 -DNEAR -Ox -g",
-                                asoutflag => "-o" };
+            $vc_win64a_info = { AS        => "nasm",
+                                ASFLAGS   => "-Ox -g",
+                                asflags   => "-f win64 -DNEAR",
```

Comment:
Even though -Ox is an optimization flag, it's a bit different with nasm, -Ox is more or less must. Normally switching off optimizations is kind of about debugging, but here your debugging options are not affected a little bit. Though I can argue that nobody will try to override this, Windows users are not into flags that much...

Reply:
Will fix.